<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathsForKids</class>
 <widget class="QMainWindow" name="MathsForKids">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MathsForKids</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(160, 160, 255);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quick Math For Kiddos </string>
    </property>
   </widget>
   <widget class="QLabel" name="messageBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 148, 241);

</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeSettingsBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="settingsLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Game Settings</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetScore">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="difficultyLabel">
       <property name="text">
        <string>Difficulty Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Difficulty">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>334</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="MultiplicationBtn">
       <property name="text">
        <string>Multiplication</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SubtractionBtn">
       <property name="text">
        <string>Subtraction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SumBtn">
       <property name="text">
        <string>Sum </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DivisionBtn_2">
       <property name="text">
        <string>Division</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="pointsLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>scores</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Score Board | </string>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Question : </string>
    </property>
   </widget>
   <widget class="QLabel" name="mathQuestion">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 255);</string>
    </property>
    <property name="text">
     <string>Text Message Box</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AnswerText">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SubmitBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skipBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
